pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [project]
 name = "kaggle-environments"
-version = "1.22.0"
+version = "1.22.15"
```

Comment:
@chris-dmello - might be easiest for you to go with version 1.23 on your P.R. :)